tests/CMakeLists.txt

```diff
@@ -152,6 +152,7 @@ if(AVIF_GTEST)
     add_avif_gtest_with_data(avifsize0test)
     add_avif_internal_gtest(avifstreamtest)
     add_avif_gtest(avifsvttest)
+    target_include_directories(avifsvttest PRIVATE ${SVT_INCLUDE_DIR}) # To be able to check the SVT-AV1 version in test.
```

Comment:
We should only do this if SVT-AV1 is used. Maybe check if `SVT_INCLUDE_DIR` is defined or nonempty?

Also, we don't need to build avifsvttest if SVT-AV1 is not used. See my comment at tests/gtest/avifsvttest.cc:37.

Reply:
Went with `if(AVIF_CODEC_SVT_ENABLED)`.